dipy/tracking/local_tracking.py

```diff
@@ -79,6 +80,9 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
             raise ValueError("step_size must be greater than 0.")
         if maxlen < 1:
             raise ValueError("maxlen must be greater than 0.")
+        if not isinstance(seeds, Iterable):
+            raise ValueError("seeds should be a list of 3D points. Please"
```

Comment:
This seems a bit confusing to me. The first sentence says the input should be a list and then the second sentence says it should be an array. I think either is fine? Could you please clarify that?

Reply:
Ok :+1: , will do